packages/amplify-graphql-default-value-transformer/src/__tests__/amplify-grapphql-default-value-transformer.test.ts

```diff
@@ -41,180 +46,35 @@ describe('DefaultValueModelTransformer:', () => {
     ).toThrow('The @default directive may only be added to scalar or enum field types.');
   });
 
-  it('throws if @default is used with a null value', () => {
+  it.each([
```

Comment:
This change can be dropped. I just felt it helped me scrolling around through this file a bunch.

Reply:
Not following your comment. This seems like a good refactor, so why should it be dropped?

Reply:
Gotcha. It's a good change, thanks for adding it!